src/components/counter-element.js

```diff
@@ -25,9 +25,8 @@ class CounterElement extends LitElement {
       <div>
         <p>
           Clicked: <span>${props.clicks}</span> times.
-          Value is <span>${props.value}</span>.
```

Comment:
Why are we not showing the value anymore? It's part of this element, regardless of what the rest of the app is showing. This element illustrates a third party element that you got off a shelf, so I'd like us to not consider that the value is shown somewhere else in the app when looking at what this element should show.

Reply:
Didn't consider that aspect, I concur.